README.md

```diff
@@ -6,4 +6,4 @@ A Python library to control ffmepg from asyncio for [Home Assistant](https://www
 - Analyse a video/audio stream for noise or motion detection
 - Grab image from a stream
 
-Be carfull that you protect function calls to this library with `asyncio.shield`.
+Be careful that you protect function calls to this library with `asyncio.shield`.
```

Comment:
The English of this sentence is still not correct. I'll check with some people if we know the intention of this sentence.

Reply:
```suggestion
Take care to protect function calls to this library with `asyncio.shield`.
```

Reply:
Yes, the better way to phrase that.